src/addon_folder.ts

```diff
@@ -157,7 +156,7 @@ async function foldersToWorkspaceFoldersMockup(folders: string[]) {
 
         mockups.push({
             "name" : path.basename(absolutePath),
-            "uri": vscode.Uri.parse(absolutePath),
+            "uri": vscode.Uri.from({ scheme: "file", path: absolutePath }),
```

Comment:
@Mateusz-Grzelinski I think this line is the most important - `scheme: "file"`, without it the URI doesn't work correctly when the addons are loaded from drive other than the extension. @kiraacorsac can confirm this, he had issues with it.

Reply:
Yes, that is the issue. Hopefully I remember this correctly, but `Uri.parse` does not behave correctly as it interprets a non-system drive letter as a protocol (`D:\path` becomes just `path` with `D` protocol. `Uri.from` makes sure the path is interpreted correctly.